e2e-tests/operator-self-healing/run

```diff
@@ -122,6 +126,8 @@ main() {
     desc 'emulate bad network'
     netem_pod
 
+    kubectl_bin apply -f "$test_dir/conf/cw-rbac-with-hook.yaml"
```

Comment:
test can run in cluster-wide and per-namespace mode, we should detect mode like in deploy_operator function.
or just run `deploy_operator` function here, instead of this command

Reply:
e61e348